distribution/docs/src/main/resources/_contents/_securing/environment-hardening-contents.adoc

```diff
@@ -62,4 +62,13 @@ a|Update the `<${branding}_HOME>/etc/org.ops4j.pax.web.cfg` file to add the entr
 Setting this configuration may break compatibility to legacy systems that do not support two-way SSL.
 ====
 
+|Session Inactivity Timeout
+|unauthorized access
+a|Update the Session configuration to have no greater than 10 minute Session Timeout.
+
+ * Navigate to the *${admin-console}*.
+ * Select the *${ddf-security}* application.
+ * Select the *Configuration* tab.
+ * Select `Session`.
+ * Set `Session Timeout (in minutes)` to `10` (or less).
```

Comment:
Can you double check how this renders in the pdf version? I'm worried you might lose the line breaks if it doesn't render a bulleted list in the table cell. You _can_ force it by adding a `+` to the ends of the lines. Approved if that looks ok.